frontend/admin/src/js/components/user/UserTableFilterDialog.tsx

```diff
@@ -0,0 +1,228 @@
+import React from "react";
+import Dialog from "@common/components/Dialog";
+import { Button } from "@common/components";
+import { FormattedMessage, useIntl } from "react-intl";
+import { BasicForm } from "@common/components/form";
+import SelectFieldV2 from "@common/components/form/Select/SelectFieldV2";
+import MultiSelectFieldV2 from "@common/components/form/MultiSelect/MultiSelectFieldV2";
+import "./UserTableFilterDialog.css";
+import { useFormContext } from "react-hook-form";
+import type { SubmitHandler } from "react-hook-form";
+import type { Option } from "@common/components/form/Select/SelectFieldV2";
+import useFilterOptions from "./useFilterOptions";
+
+export type FormValues = {
+  pools: Option["value"][];
+  languageAbility: Option["value"][];
+  classifications: Option["value"][];
+  operationalRequirement: Option["value"][];
+  workRegion: Option["value"][];
+  // TODO: Make mandatory once data model settles.
+  // See: https://www.figma.com/proto/XS4Ag6GWcgdq2dBlLzBkay?node-id=1064:5862#224617157
+  educationType?: Option["value"][];
+  employmentDuration: Option["value"][];
+  jobLookingStatus: Option["value"][];
+  skills: Option["value"][];
+  profileComplete: Option["value"][];
+  govEmployee: Option["value"][];
+};
+
+type FooterProps = Pick<UserTableFilterDialogProps, "enableEducationType">;
+const Footer = ({ enableEducationType }: FooterProps): JSX.Element => {
+  const { reset } = useFormContext();
+  const { emptyFormValues } = useFilterOptions(enableEducationType);
+  const handleClear = () => {
+    reset(emptyFormValues);
```

Comment:
This is what makes "Clear filter" clear the forms to empty, rather than revert to them to `activeFilters` state. It we wanted "Clear" to instead revert to `activeFilters` state, we would prop drill `activeFilters` into the footer and maybe rename the button.

Reply:
I prefer what you did here. I would assume a "clear" button resets everything to a null state rather than the previous state.